scripts/configure-installer-tests-cluster.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+# Setup to find nessasary data from cluster setup
+## Constants
+HTPASSWD_FILE="./htpass"
+USERPASS="developer"
+HTPASSWD_SECRET="htpasswd-secret"
+# Defaults
+DEFAULT_INSTALLER_ASSETS_DIR=$(pwd)
```

Comment:
one last thing 😇 can we make this also overrideable?

Reply:
It already is, you can directly set $KUBECONFIG and $KUBEADMIN_PASSWORD_FILE to override this :)